templates/base.html

```diff
@@ -19,8 +19,10 @@ <h3 class="text-center">A website with information about automobiles for buying/
     {% block content %}{% endblock content %}
     </body>
 
-<script src="{% static 'js/jquery-3.2.1.slim.min.js' %}"></script>
+<!-- <script src="{% static 'jquery-3.2.1.slim.min.js' %}"></script> -->
+<script src="{% static 'js/jquery-3.1.0.min.js' %}"></script>
```

Comment:
we previously had the slim version of jQuery which does not support AJAX.
https://stackoverflow.com/questions/44212202/my-javascript-is-returning-this-error-ajax-is-not-a-function